resources/qml/WelcomePages/PrinterCard.qml

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2022 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+
+import UM 1.5 as UM
+import Cura 1.1 as Cura
+
+Control
+{
+    id: root
+    property alias text: link_text.text
+    property alias imageSource: image.source
+    property var onClicked
+
+    states:
+    [
+        State
+        {
+            name: "hovered";
+            when: mouse_area.containsMouse
+            PropertyChanges
+            {
+                target: background
+                color: UM.Theme.getColor("monitor_card_hover")
+            }
+            PropertyChanges
+            {
+                target: link_text
+                font.underline: true
+            }
+        }
+    ]
+
+    MouseArea
+    {
+        id: mouse_area
+        anchors.fill: parent
+        hoverEnabled: true
+        onClicked: root.onClicked && root.onClicked()
```

Comment:
Does root.onClicked do something here?

Reply:
Its guarding against the property not being set